### PR TITLE
Release mcan/0.4.0

### PR DESCRIPTION
**Pull request by Ironedde**
### Added
 - Add `Can::aux::initialization_mode` (#41)

### Changed
 - Fix some issues with watermark sizes for Rx FIFOs and Tx Event FIFO (#43)
 - Adhere to `filter_map_bool_then` clippy lint (#42)

### CODE REVIEW

**Comment by Ironedde** [comment]
**TODO after merge:**

- [ ] Add tag `mcan/0.4.0` to the master branch 
- [ ] Publish to crates.io

**Review by epontan — CHANGES_REQUESTED** [review]
Looks good, just minor nit in the changelog.

File: mcan/CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+## [0.4.0] - 2023-10-24
+
+### Added
 - Add `Can::aux::initialization_mode` (#41)
+
+### Changed
+- Fix some issues with watermark sizes for Rx FIFOs and Tx Event FIFO (#43)
 - Adhere to `filter_map_bool_then` clippy lint (#42)
 
+
```
[the remark this answers is not in the document]
Comment (Ironedde) [reply mcan/CHANGELOG.md]: Ah, must have copied one line to much when moving from Unreleased section.
Fixed it :+1:

**Review by epontan — APPROVED** [review]
LGTM :+1:

**Pull request merged**